### PR TITLE
New version: UweSieber.UsbTreeView version 4.3.2

### PR DESCRIPTION
**Pull request by i0ntempest**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/166167)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-08-04 23:19:54.468 [FAIL] Installer failed security check. Url: https://www.uwe-sieber.de/files/UsbTreeView_x64.zip Result: 0x80004005

Automated error analysis suggests -1978335187 may mean Installer failed security check. Another script should retry this every 18 hours. 

(Automated response - build 887.)

**Comment by i0ntempest** [comment]
What does this mean? Defender detecting the installer as a virus?

**Comment by stephengillie** [comment]
> 8/6/2024 1:26:16 AM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   1F18CF5BC3E49A0C9A2555CC353F7988534C6B3D834F5D0233912FCD35B117CB
Expected hash: a99a9ee97a8f64fb275c470b9a947aaefbd20a76081fc083d37e2c1a7c5f0d81

File: manifests/u/UweSieber/UsbTreeView/4.3.2/UweSieber.UsbTreeView.installer.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@ UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerUrl: https://www.uwe-sieber.de/files/UsbTreeView_x64.zip
-  InstallerSha256: 1BC38D30FEE3C387A484478CFB146770CF4462D6104326614E5F60F59F1F6D8C
+  InstallerSha256: A99A9EE97A8F64FB275C470B9A947AAEFBD20A76081FC083D37E2C1A7C5F0D81
```
Comment (stephengillie) [inline manifests/u/UweSieber/UsbTreeView/4.3.2/UweSieber.UsbTreeView.installer.yaml:14]:
```suggestion
  InstallerSha256: 1F18CF5BC3E49A0C9A2555CC353F7988534C6B3D834F5D0233912FCD35B117CB
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @i0ntempest,

The hash has updated - has the developer released a newer version?

**Pull request closed without merging**